airflow/www/views.py

```diff
@@ -1921,23 +1972,22 @@ def failed(self):
     @action_logging
     def success(self):
         """Mark task as success."""
-        dag_id = request.form.get('dag_id')
-        task_id = request.form.get('task_id')
-        origin = get_safe_url(request.form.get('origin'))
-        execution_date = request.form.get('execution_date')
+        args = request.form
+        dag_id = args.get('dag_id')
+        task_id = args.get('task_id')
+        origin = get_safe_url(args.get('origin'))
+        execution_date = args.get('execution_date')
 
-        confirmed = request.form.get('confirmed') == "true"
-        upstream = request.form.get('success_upstream') == "true"
-        downstream = request.form.get('success_downstream') == "true"
-        future = request.form.get('success_future') == "true"
-        past = request.form.get('success_past') == "true"
+        upstream = to_boolean(args.get('failed_upstream'))
```

Comment:
This looks like a bug, why is the options `failed_upstream`? It should be `success_upstream`. Same for the other args. @msumit could you fix?

Reply:
Thanks for catching this @yuqian90.. opened the fix for this -> https://github.com/apache/airflow/pull/16524